public/hack.js

```diff
@@ -127,6 +127,12 @@ uiModules.get('app/dashboard', []).run(function ($http, $location, kbnUrl, getAp
                 getAppState().save();
                 return;
 
+            case "flushSearchChip":
+                for(let i = 0; i < getAppState().filters.length; i++) {
+                    getAppState().filters.pop();
+                }
+                return;
```

Comment:
Instead the for loop simple add the next line:
`                getAppState().filters = [];
`